lib/cli/src/generators/generator.ts

```diff
@@ -50,9 +50,20 @@ const generator = async (
     framework !== 'angular' ? '@storybook/addon-essentials' : '@storybook/addon-actions',
   ];
 
-  const packages = [`@storybook/${framework}`, ...addons, ...extraPackages, ...extraAddons].filter(
-    Boolean
-  );
+  // ⚠️ Some addons have peer deps that must be added too, like '@storybook/addon-docs' => 'react-is'
+  const addonsPeerDeps = addons.some(
+    (addon) => addon === '@storybook/addon-essentials' || addon === '@storybook/addon-docs'
+  )
+    ? ['react-is']
+    : [];
```

Comment:
@shilman any reason to have `react-is` as a peerDep and not a regular dep? I'm guessing because having multiples different versions in a single project leads to chaos💥?